### ISSUE TITLE
Feedback about torch.addcdiv

### ISSUE THREAD

**Issue by mcw-anasuya** [pytorch/pytorch#170869]
There is the following issue on this page: https://docs.pytorch.org/docs/stable/generated/torch.addcdiv.html
The description states that addcdiv performs the element-wise division of tensor1 by tensor2, multiplies the result by the scalar value and adds it to input.
According to this, the formula of addcdiv should be: `input + value * (tensor_1/tensor2)`

Under keyword arguments in the documentation, its also mentioned that `value (Number, optional) – multiplier for tensor1/tensor2` which implies `tensor1/tensor2` is computed first.
However, its clear from evaluation that `value*tensor1` is being computed first.

So the actual formula torch.addcdiv follows is: `input + (value * tensor_1)/tensor2`

Test to Reproduce:
```
import torch

# Input values
value_a = 1.00000001335143e-10
value_b = -2.84726048044011e38
value_c = -2.84726048044011e38
scalar_value = 2.75

# Create float32 tensors
A = torch.tensor(value_a, dtype=torch.float32)
B = torch.tensor(value_b, dtype=torch.float32)
C = torch.tensor(value_c, dtype=torch.float32)

# Compute addcdiv
out = torch.addcdiv(A, B, C, value=scalar_value)

# Print detailed results
print("A:", A.item())
print("B:", B.item())
print("C:", C.item())
print("B / C:", (B / C).item())
print("scalar * (B / C):", (scalar_value * (B / C)).item())
print("torch.addcdiv output:", out.item())
print("Is output inf?", torch.isinf(out).item())
```

Similar results are found in torch.addcmul

**Comment by malfet** [comment]
I don't think that such clarification to the documentation would help people in any way or if there is a good solution on how to achieve best accuracy without sacrificing too much perf

**Comment by jiren-the-gray** [comment]
This mismatch also exists on the `torch.addcmul` documentation, which states that the order is `(t1*t2) * value + input`, but on CPU it is clearly calculated as `(t1*value) * t2 + input`. This can have unexpected situations, where the `value` is large enough to cause `t1*value` to overflow, but not `(t1*t2) * value` to overflow. In the [gist](https://colab.research.google.com/gist/jiren-the-gray/2e6c8dabac51726af8a482129fdbc995/torch-addcmul.ipynb) here, I have shown this scenario. If the documentation and the CPU implementation is in sync, it will be easier to debug cases like this IMO.
